### ISSUE TITLE
Implement general layout redesign

### ISSUE THREAD

**Issue by davidsandoz** [veganaut/veganaut#225]
Implementation of the redesign of the common parts of the app, which is basically the header.

**Comment by davidsandoz** [comment]
Header redesign is done. I'm ready to merge it to the `release-1-0-0-space-chick` branch. But as I merged the `refactor-location` branch into the `header-redesign` branch, it'd be better to first merge `refactor-location` into `release-1-0-0-space-chick` before doing so with `header-redesign`.

Do you know if that would happen soon @renestalder?

**Comment by renestalder** [comment]
Regarding conflicts: It doesn't matter. You already have my changes, so no conflicts occur between my branch, the release branch and your branch. And I frequently merge the main branch in my branch. So everything's fine here.

Regarding my work that isn't finished, but in your branch now: It doesn't break the build. The location details page just isn't complete, but as long we're not going to release parts of the release branch in advance of finishing all the features, that's not a problem. I'll proceed soon and finish my part in my branch to complete the release branch.

**Comment by renestalder** [comment]
btw. short note: You don't have to reference every commit with the ticket like we do for Trac. Usually it makes sense only adding the reference in the Merge commit / Pull request to not overflow the issue. That's really just a rule we use for Trac, but in Gitlab / Github environment with Merge Requests it's now much more easier to keep track without permanent references.

I referenced several commits with this ticket here to leave a note and notify you that I touched parts of your task.
